samples/videoDecodeBasic/README.md

```diff
@@ -0,0 +1,29 @@
+# Video decode sample
+
+The video decode baisc sample illustrates decoding a single packetized video stream using the built-in bitstream reader, video parser, and rocDecoder to get the individual decoded frames in YUV format. This sample can be configured with a device ID and optionally able to dump the output to a file. This sample uses the high-level RocVideoDecoder class which connects both the video parser and Rocdecoder. This process repeats in a loop until all frames have been decoded.
+
+## Prerequisites:
+
+* Install [rocDecode](../../README.md#build-and-install-instructions)
+
+## Build
+
+```shell
+mkdir video_decode_sample && cd video_decode_sample
+cmake ../
+make -j
+```
+
+## Run
+
+```shell
+./videodecode -i <input video file [required]> 
```

Comment:
this would change to `./videodecodebasic` -- based on your executable in CMakeLists

Reply:
Done.